data/json/items/ammo/20x66mm.json

```diff
@@ -33,50 +33,50 @@
-    "description": "Handcrafted bootleg duplicates of Rivtech 20x66mm caseless flechette rounds.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "description": "Handcrafted bootleg duplicates of Rivtech 20x66mm caseless flechette rounds.",
     "proportional": { "price": 0.7, "shot_damage": { "damage_type": "bullet", "amount": 0.9 }, "dispersion": 1.1 },
-    "extend": { "effects": [ "RECYCLED" ] },
+    "extend": { "effects": [ "RECYCLED" ], "flags": [ "CASELESS_ROUNDS" ] },
     "delete": { "effects": [ "NEVER_MISFIRES" ], "flags": [ "IRREPLACEABLE_CONSUMABLE" ] }
   },
   {
     "id": "20x66_bootleg_shot",
     "copy-from": "20x66_shot",
     "type": "AMMO",
     "name": { "str_sp": "20x66mm buckshot, handmade" },
-    "description": "Handcrafted bootleg duplicates of Rivtech 20x66mm caseless buckshot rounds.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "description": "Handcrafted bootleg duplicates of Rivtech 20x66mm caseless buckshot rounds.",
     "proportional": { "price": 0.7, "shot_damage": { "damage_type": "bullet", "amount": 0.9 }, "shot_spread": 1.1 },
-    "extend": { "effects": [ "RECYCLED" ] },
+    "extend": { "effects": [ "RECYCLED" ], "flags": [ "CASELESS_ROUNDS" ] },
     "delete": { "effects": [ "NEVER_MISFIRES" ], "flags": [ "IRREPLACEABLE_CONSUMABLE" ] }
   },
   {
     "id": "20x66_bootleg_slug",
     "copy-from": "20x66_slug",
     "type": "AMMO",
     "name": { "str": "20x66mm slug, handmade", "str_pl": "20x66mm slugs, handmade" },
-    "description": "Handcrafted bootleg duplicates of Rivtech 20x66mm caseless solid projectile rounds.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "description": "Handcrafted bootleg duplicates of Rivtech 20x66mm caseless solid projectile rounds.",
     "proportional": { "price": 0.7, "damage": { "damage_type": "bullet", "amount": 0.9 }, "dispersion": 1.1 },
-    "extend": { "effects": [ "RECYCLED" ] },
+    "extend": { "effects": [ "RECYCLED" ], "flags": [ "CASELESS_ROUNDS" ] },
     "delete": { "effects": [ "NEVER_MISFIRES" ], "flags": [ "IRREPLACEABLE_CONSUMABLE" ] }
   },
   {
     "id": "20x66_exp",
     "copy-from": "20x66_abstract",
     "type": "AMMO",
     "name": { "str_sp": "20x66mm explosive" },
-    "description": "20x66mm caseless shotgun rounds, explosive projectile type.  Proprietary ammunition for Rivtech shotguns.  Being caseless rounds, these cannot be disassembled or reloaded.",
+    "description": "20x66mm caseless shotgun rounds, explosive projectile type.  Proprietary ammunition for Rivtech shotguns.",
     "price": 10000,
     "price_postapoc": 8000,
     "range": 20,
     "damage": { "damage_type": "bullet", "amount": 16, "armor_penetration": 4 },
     "dispersion": 120,
-    "effects": [ "COOKOFF", "EXPLOSIVE_20x66", "NEVER_MISFIRES" ]
+    "extend": { "effects": [ "EXPLOSIVE_20x66" ], "flags": [ "CASELESS_ROUNDS" ] }
```

Comment:
I just realized it; was removal of `COOKOFF` and `NEVER_MISFIERES` flag intentional @haveric?

Reply:
It's extending the abstract which includes those: https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/items/ammo/20x66mm.json#L15